Source/WebKit/Shared/CoordinatedGraphics/CoordinatedGraphicsScene.cpp

```diff
@@ -37,8 +37,9 @@
 namespace WebKit {
 using namespace WebCore;
 
-CoordinatedGraphicsScene::CoordinatedGraphicsScene(CoordinatedGraphicsSceneClient* client)
+CoordinatedGraphicsScene::CoordinatedGraphicsScene(CoordinatedGraphicsSceneClient* client, bool recordDamage)
```

Comment:
Same here, better to use an enum and rename it as shouldRecordDamage

Reply:
Done.